src/test/java/core/basesyntax/DataConvertorImplTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConvertor;
+import core.basesyntax.service.impl.DataConvertorImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class DataConvertorImplTest {
+    private DataConvertor dataConvertor;
+    private List<String> lines;
+    private List<FruitTransaction> fruitTransactions;
+
+    @BeforeEach
+    void testUp() {
+        dataConvertor = new DataConvertorImpl();
+        lines = new ArrayList<>();
+    }
+
+    @Test
+    void how_data_convertor_works() {
```

Comment:
use some name pattern for test cases.
`methodName_parameters_expectedBehavior`
```suggestion
    void dataConvert_validInputData_ok() {
```
Fix it everywhere